include/open62541/plugin/pki.h

```diff
@@ -52,7 +52,10 @@ struct UA_CertificateVerification {
     /* Delete the certificate verification context */
     void (*clear)(UA_CertificateVerification *cv);
 
-    const UA_Logger *logger;
+    /* Pointer to logger pointer
```

Comment:
Rename to `logging`. Describe as Pointer to logging pointer in the server/client configuration.

Reply:
renamed